cMake/FreeCAD_Helpers/SetupShibokenAndPyside.cmake

```diff
@@ -43,6 +43,18 @@ macro(SetupShibokenAndPyside)
             endif()
         endif()
 
+        # pyside2 changed it's cmake files, this is the dance we have
+        # to dance to be compatible with the old (<5.12) and the new versions (>=5.12)
+        if(NOT SHIBOKEN_LIBRARY AND TARGET Shiboken2::libshiboken)
```

Comment:
testing for imported target Shiboken2::libshiboken doesn't work here. Any ideas how to check for an imported target?